vm/src/stdlib/random.rs

```diff
@@ -1,15 +1,82 @@
 //! Random module.
 
+use std::cell::RefCell;
+
+use num_bigint::{BigInt, Sign};
+
 use rand::distributions::Distribution;
+use rand::{RngCore, SeedableRng};
+use rand::rngs::SmallRng;
 use rand_distr::Normal;
 
-use crate::pyobject::{PyObjectRef, PyResult};
+use crate::function::OptionalArg;
+use crate::obj::objtype::PyClassRef;
+use crate::pyobject::{PyClassImpl, PyObjectRef, PyRef, PyValue, PyResult};
+
 use crate::vm::VirtualMachine;
 
+#[pyclass(name = "Random")]
+#[derive(Debug)]
+struct PyRandom {
+    rng: RefCell<SmallRng>
```

Comment:
I would strongly recommend to use `StdRng` as a default instead, it is a much safer choice, because it is not predictable.

Reply:
It would probably be possible to have an enum with both `StdRng` and `SmallRng` as variants, and only switch to `SmallRng` when `seed` is called.

Reply:
`SmallRng` is not the Mersenne Twister, so it does not make sense to switch to it for compatibility with CPython. Other than that, I agree!